src/order-book/generated/models/OrderQuoteRequest.ts

```diff
@@ -45,10 +45,6 @@ export type OrderQuoteRequest = (OrderQuoteSide & OrderQuoteValidity & {
      *
      */
     appDataHash?: AppDataHash;
-    /**
-     * Is the order fill-or-kill or partially fillable?
-     */
-    partiallyFillable?: boolean;
```

Comment:
Is this gone from the quote request??

Reply:
Yep, seems so, see: https://github.com/cowprotocol/services/pull/2362